src/daemon/ca/manager.rs

```diff
@@ -500,8 +500,10 @@ impl CaManager {
 
     /// Adds the associated signer to the proxy.
     ///
-    /// Errors if there is no proxy or the proxy already has a signer.
-    pub fn ta_proxy_signer_add(
+    /// Errors if:
+    /// - there is no proxy
+    /// - the proxy has a signer
+    pub async fn ta_proxy_signer_add(
```

Comment:
The method doesn’t need to be async, so it shouldn’t be. (Also, maybe keep the new docstring)?